src/app/RootNavigator.tsx

```diff
@@ -42,10 +45,33 @@ export const RootNavigator = () => {
           <IonList>
             <IonMenuToggle>
               <UserMenu role={auth.user?.role} />
-              <IonItem>
-                <IonButton onClick={handleLogout}>Log out</IonButton>
-              </IonItem>
             </IonMenuToggle>
+            {auth.user?.role === "FARMER" ? (
+              <IonItem lines="none">
+                <IonMenuToggle>
+                  <IonButton
+                    href="https://wa.me/?text=Hi%2C%20please%20visit%20my%20farm.%20My%20chickens%20are%20sick.%20"
```

Comment:
Can you hide this?

Reply:
the `href`? It should be the same for any env so there's no need